manifests/capacityscheduling/crd.yaml

```diff
@@ -1,4 +1,5 @@
 
+{{- if has "capacityScheduling" .Values.plugins.enable}}
```

Comment:
I think this will fail at https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/hack/verify-crdgen.sh.

Can we find another way to keep this yaml intact?

Reply:
It will work if I removed it but the crd will be deployed even we don't explicitly add it to the enable list. what do you think?

Reply:
Given these crds serve multiple purposes. I think removing the helm directives outweigh the benefits it brings in. So yes, I'm fine with registering the CRDs even if the corresponding plugin is not enabled.